packages/node_modules/@webex/internal-plugin-devices/src/feature-collection.js

```diff
@@ -0,0 +1,30 @@
+// External dependencies.
+import AmpCollection from 'ampersand-collection';
+
+// Internal dependencies.
+import FeatureModel from './feature-model';
+
+/**
+ * Feature collection model.
+ *
+ * @description
+ * This model contains a collection of features under a specific collection
+ * group.
+ */
+const FeatureCollection = AmpCollection.extend({
+  /**
+   * The unique identifier for the models in this collection.
+   *
+   * @type {string}
+   */
+  mainIndex: 'key',
+
+  /**
+   * The type of model this collection can contain.
+   *
+   * @type {Class}
+   */
+  model: FeatureModel
+});
+
+export default FeatureCollection;
```

Comment:
Since this file is simply a collections file, not sure if unit/integration tests are needed.

Reply:
Basic unit tests would be good, you can use the meetings collection tests as a reference:
https://github.com/webex/webex-js-sdk/blob/master/packages/node_modules/@webex/plugin-meetings/test/unit/spec/meetings/collection.js#L34